languages.toml

```diff
@@ -481,7 +481,7 @@ indent = { tab-width = 2, unit = "  " }
 
 [[grammar]]
 name = "nix"
-source = { git = "https://github.com/cstrahan/tree-sitter-nix", rev = "6b71a810c0acd49b980c50fc79092561f7cee307" }
+source = { git = "https://github.com/cstrahan/tree-sitter-nix", rev = "1b69cf1fa92366eefbe6863c184e5d2ece5f187d" }
```

Comment:
updating the grammar seems orthogonal to this PR. Considering there is a conflict here this has been bumped on master already anyway.

Reply:
Actually there was a PR that fixed an issue with Nix highlighting included here. After looking at the pinned commit in master, for some reason it was pinned before my PR was merged, so a bump is still in order or one of the queries won't even function.